lib/LaTeXML/Package/acmart.cls.ltxml

```diff
@@ -73,16 +73,18 @@ DefMacro('\subtitle{}',            '\@add@frontmatter{ltx:subtitle}{#1}');
 DefMacro('\keywords{}',            '\@add@frontmatter{ltx:keywords}{#1}');
 DefMacro('\terms{}',               '\@add@frontmatter{ltx:keywords}{#1}');
 
-# Accessible figure descriptions
-NewCounter('acmlabel');
-DefConstructor('\Description[]{}', '^^<ltx:note xml:id="#id" class="ltx_nodisplay">#1</ltx:note>',
-  properties => sub { ('width' => Dimension(0), 'height' => Dimension(0), RefStepCounter('acmlabel')) },
-  beforeConstruct => sub {
-    my ($document, $whatsit) = @_;
-    # TODO: Is there something useful to do with the short description in our schema?
-    my $figure = $document->getElement;
-    $document->setAttribute($figure, 'aria:labelledby', $whatsit->getProperty('id'));
-    return; });
+# Accessible figures: \Description[alternate text]{long descriptions}
+DefConstructor('\Description[]{}', sub {
+  my ($document, $alt, $desc) = @_;
+  # Could this apply to other than figure?
+  if (my $savenode = $document->floatToElement('ltx:figure')) {
+    $document->insertElement('ltx:alternate-text',   $alt)  if defined $alt;
+    $document->insertElement('ltx:long-description', $desc) if defined $desc;
```

Comment:
In practice, often the author provides only one of the two arguments, in which case it is most predictable for us to be setting the primary accessible description only (i.e. ltx:alternate-text). It is also the field requires by accessibility checkers, the long-description is always optional.

So it is unfortunate the optional argument is used for the mandatory field for remediation.